File: Week 01/id_662/26.remove-duplicates-from-sorted-array.java
class Solution {
    public static int removeDuplicates(int[] nums) {
        int length=0;
        for(int i=1;i<nums.length;i++){
            if(nums[length]!=nums[i]){
                nums[++length] = nums[i];
            }
        }
        return length+1;
    }
}

File: Week 01/id_682/26.remove-duplicates-from-sorted-array.php
<?php 

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        newLen =  1
        
        if len(nums) == 0:
            return 0
        
        for i in range(1, len(nums)) :
            if nums[i] != nums[i-1]:
                nums[newLen] = nums[i]
                newLen += 1
        return newLen
            